sample/dll.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="icon" type="image/png" href="https://ckeditor.com/docs/ckeditor5/latest/assets/img/favicons/32x32.png" sizes="32x32">
+    <meta charset="utf-8">
+    <title>CKEditor 5 with ckeditor5-math – DLL Sample</title>
+    <link
+        rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/katex@0.13.5/dist/katex.min.css"
+        integrity="sha384-L+Gq2Cso/Y2x8fX4wausgiZT8z0QPZz7OqPuz4YqAycQJyrJT9NRLpjFBD6zlOia"
+        crossorigin="anonymous"
+    />
+    <script
+        src="https://cdn.jsdelivr.net/npm/katex@0.13.5/dist/katex.min.js"
+        integrity="sha384-z64WtjpyrKFsxox9eI4SI8eM9toXdoYeWb5Qh+8PO+eG54Bv9BZqf9xNhlcLf/sA"
+        crossorigin="anonymous"
+    ></script>
+    <script
+        src="https://cdn.jsdelivr.net/npm/katex@0.13.5/dist/contrib/auto-render.min.js"
```

Comment:
About the `Uncaught Error: No element provided to render` error that happens with Katex, I think it can be solved just by adding the `defer` attr to the script as described in [the doc](https://katex.org/docs/autorender.html#usage).

```suggestion
    <script
        defer
        src="https://cdn.jsdelivr.net/npm/katex@0.13.5/dist/contrib/auto-render.min.js"
```

Reply:
I had to remove this to prevent load errors first reported.

Reply:
But, hey ... feel free to try and report anything that works - I'm not really familiar with Katex, as I am using MathJax in my project.